--- a/Mailman/Handlers/Hold.py
+++ b/Mailman/Handlers/Hold.py
@@ -221,7 +221,10 @@
     # translator again, because of the games we play above
     reason = Utils.wrap(exc.reason_notice())
     msgdata['rejection_notice'] = Utils.wrap(exc.rejection_notice(mlist))
-    id = mlist.HoldMessage(msg, reason, msgdata)
+    spamscore = ''
+    if 'spamscore' in msgdata:
+        spamscore = ' (spam score %s)' % msgdata['spamscore']
+    id = mlist.HoldMessage(msg, reason + spamscore, msgdata)
     # Now we need to craft and send a message to the list admin so they can
     # deal with the held message.
     d = {'listname'   : listname,
